docs/reference/commandline/attach.md

```diff
@@ -51,16 +52,16 @@ While a client is connected to container's `stdio` using `docker attach`, Docker
 uses a ~1MB memory buffer to maximize the throughput of the application.
 Once this buffer is full, the speed of the API connection is affected, and so
 this impacts the output process' writing speed. This is similar to other
-applications like SSH. Because of this, it is not recommended to run
-performance critical applications that generate a lot of output in the
-foreground over a slow client connection. Instead, users should use the
+applications like SSH. Because of this, it isn't recommended to run
+performance-critical applications that generate a lot of output in the
+foreground over a slow client connection. Instead, use the
 `docker logs` command to get access to the logs.
 
 ## Examples
 
 ### Attach to and detach from a running container
 
-The following example starts an ubuntu container running `top` in detached mode,
+The following example starts an Ubuntu container running `top` in detached mode,
```

Comment:
Heh. Probably unrelated, but we could use `alpine` (or `busybox`) for the example. The `/usr/bin/top` is also a bit awkward (it's in `PATH` so no need to specify the full path)